packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -81,23 +122,29 @@ const BountySummary = ({
 	);
 };
 
-const BountyDetails = ({
-	_id,
-	description,
-	criteria,
-	createdBy,
-	claimedBy,
-	status,
-	discordMessageId,
-}: BountyCollection): JSX.Element => {
+const BountyDetails = ({ bounty }: { bounty: BountyCollection }): JSX.Element => {
+	const {
+		_id,
+		description,
+		criteria,
+		createdBy,
+		claimedBy,
+		status,
+		discordMessageId,
+		dueAt,
+	} = bounty;
 	const { customer: { customer_id, bountyChannel } } = useContext(CustomerContext);
-	const url = discordMessageId ? `${baseUrl}/${customer_id}/${bountyChannel}/${discordMessageId}` : '/';
+	const url = discordMessageId
+		? `${baseUrl}/${customer_id}/${bountyChannel}/${discordMessageId}`
+		: `${baseUrl}/${customer_id}/${bountyChannel}`;
 	return (
 		<Grid gap={6}>
-			<GridItem>
-				<Heading size="sm">HashID</Heading>
-				<Text>{_id}</Text>
-			</GridItem>
+			{ _id &&
+				<GridItem>
+					<Heading size="sm">HashID</Heading>
```

Comment:
On the bot we've migrated away from HashId to `Bounty Id`, I think sensible to do on our web client as well

Reply:
Will do moving forward